sapl/sessao/views.py

```diff
@@ -529,6 +529,190 @@ def get_presencas_generic(model, sessao, legislatura):
             yield (m.parlamentar, False)
 
 
+def filtra_materias_copia_sessao_ajax(request):
+    title_flag =  request.GET['title_flag']
+    sessao_plenaria = request.GET['sessao_plenaria_atual']
+    sessao_plenaria_destino = request.GET['sessao_plenaria_destino']
+
+    if title_flag == "Cópia de Matérias do Expediente":
+        materias_sessao_destino = [
+            expediente.materia for expediente in ExpedienteMateria.objects.filter(
+                sessao_plenaria=sessao_plenaria_destino
+            )
+        ]
```

Comment:
```suggestion
        materias_sessao_destino = ExpedienteMateria.objects.filter(sessao_plenaria=sessao_plenaria_destino).values_list('materia__id')
```

Substituir por as linhas 538-542 pelas linhas acima.